.gitignore

```diff
@@ -33,11 +33,8 @@ Vagrantfile
 /certs/saml.crt
 /config/application.yml
 /config/aws.yml
-/config/database.yml
 /config/newrelic.yml
-/config/saml.key.enc
```

Comment:
This and `secrets.yml` are not related to database.yml. They were removed from gitignore, but they weren't checked into the repo, causing `git pull` to see 2 new files added.